app/models/couch.rb

```diff
@@ -0,0 +1,4 @@
+class Couch < ApplicationRecord
+  validates_presence_of :name, :street_address, :city, :state, :zipcode
```

Comment:
@ski-climb and I talked about making names a fun feature of the app (e.g., Big Bertha). If we go with that, maybe make uniqueness true on name?